frontend/app/views/top_containers/bulk_operations/_results.html.erb

```diff
@@ -52,44 +46,46 @@
             <%= radio_button_tag "linker-item", container_json["uri"], false, :"data-object" => container_json.to_json %>
           <% end %>
         </td>
-        <td class="top-container-collection">
-          <% if doc['collection_identifier_stored_u_sstr'] %>
-            <ul class="linked-records-listing count-<%= Array(doc['collection_identifier_stored_u_sstr']).length %>">
-              <% Array(doc['collection_identifier_stored_u_sstr']).zip(Array(doc['collection_display_string_u_sstr'])).each do |identifier, display| %>
-                <li><span class="collection-identifier"><%= identifier %></span>, <span class="collection-display-string"><%= display %></span></li>
+        <% @pref_cols.each do |field| %>
+          <% if field == 'context' %>
+            <td class="top-container-collection">
+              <% if doc['collection_identifier_stored_u_sstr'] %>
+                <ul class="linked-records-listing count-<%= Array(doc['collection_identifier_stored_u_sstr']).length %>">
+                  <% Array(doc['collection_identifier_stored_u_sstr']).zip(Array(doc['collection_display_string_u_sstr'])).each do |identifier, display| %>
+                    <li><span class="collection-identifier"><%= identifier %></span>, <span class="collection-display-string"><%= display %></span></li>
+                  <% end %>
+                </ul>
               <% end %>
-            </ul>
-          <% end %>
-        </td>
-        <td class="top-container-series">
-          <% if doc['series_level_u_sstr'] %>
-            <ul class="linked-records-listing count-<%= Array(doc['series_level_u_sstr']).length %>">
-              <% Array(doc['series_level_u_sstr']).zip(Array(doc['series_identifier_stored_u_sstr'])).each do |level, identifier| %>
-                <li><span class="series-level"><%= level %></span> <span class="series-identifier"><%= identifier %></span></li>
+            </td>-=
```

Comment:
```suggestion
            </td>
```

Reply:
oof, not sure how that got in there, thanks